Automattic/woocommerce-payments
# Enable TypeScript

Fixes #1248

This PR enables TypeScript, and sets up tests, linting, and formatting to work with TS. No code is migrated in this PR.

#### Changes proposed in this Pull Request

* Enable TypeScript without migrating any code.

#### Testing instructions

* `npm ci`
* Rebuild: `npm run dev` (or `npx webpack` or `npm run watch`).
* Make sure tests run: `npm run test:js`.
* Make sure UI works: try opening the different pages in your browser and check to see if there is anything broken.

-------------------
